packages/thirdweb/src/react/web/ui/ConnectWallet/screens/Buy/fiat/currencies.tsx

```diff
@@ -2,23 +2,24 @@ import { RadiobuttonIcon } from "@radix-ui/react-icons";
 import { CADIcon } from "../../../icons/currencies/CADIcon.js";
 import { EURIcon } from "../../../icons/currencies/EURIcon.js";
 import { GBPIcon } from "../../../icons/currencies/GBPIcon.js";
+import { JPYIcon } from "../../../icons/currencies/JPYIcon.js";
 import { USDIcon } from "../../../icons/currencies/USDIcon.js";
```

Comment:
I have a list of 80+ more fiat currencies. Do we have to manually add all of the icons? What's your process for generating these?